content/reference/wadm/api.md

```diff
@@ -0,0 +1,218 @@
+
+If a consumer wishes to monitor events in real-time (for example, you want to be notified as soon as compensating actions are taken), then it can subscribe to the following topic:
+
+```
+wadm.evt.{lattice-id}
+```
+
+The events on this topic are JSON-encoded [CloudEvents](https://cloudevents.io/).
+
+**NOTE** that all model _names_ are treated like unique identifiers and must conform to the rules governing NATS topic segments. For example, they cannot contain spaces, commas, unprintable characters, or periods.
+
+## Model Persistence
+The following operations pertain to storing and retrieving models. Persistence of models is explicitly and deliberately separated from deployment and deployment management.
+
+
+### Get Model List
+`wadm.api.{lattice}.model.list`
+
+Retrieves a list of models within the given lattice. The status of the model is an aggregate of the statuses of the traits defined within. Only the newest version of the model is included in this list.
+
+**Request**: `none`
+
+**Response**: 
+```json
+{
+    "name": "..",
+    "version": "..",
+    "description": "..",
+    "deployed": true,
+    "status": "compensating|ready|failed|n/a"
+}
+```
+Undeployed models will not provide meaningful data in the `status` field.
+
+### Get Model Spec
+`wadm.api.{lattice}.model.info.{name}`
+
+Retrieves the specification of the model stored with the given name. 
+
+**Request**: `none` 
+
+**Response**:
+
+JSON serialization of [OAM model/yaml](https://github.com/wasmCloud/wadm/tree/main/oam)
+
+
+
+### Store Models
+`wadm.api.{lattice}.model.put.{name}`
+
+Stores the named model. If the model and version being submitted already exist, the request will be rejected. New versions will be appended to the history according to retention policy. Note that this won't automatically deploy a model, it only affects storage.
+
+**Request**: JSON serialization of OAM model
+
+**Response**:
+```json
+{
+    "result": "error|created|newversion",
+    "message": "..."
+}
+```
+
+The response will indicate if storage failed, if a new model was created, or if a new version of a model was pushed.
+
+
+### Delete Models
+`wadm.api.{lattice}.model.del.{name}`
+
+Deletes the named model. If a version is supplied, only that version is deleted. If no version is supplied, all versions of the model are deleted. Deleted models are immediately un-deployed if applicable. If the operation does result in an (asynchronous) attempt to undeploy, it will be indicated in the `undeploy` field.
+
+**Request**: empty, or string containing version
+
+**Response**:
+```json
+{
+    "result": "deleted|error",
+    "undeploy": true,
+    "message": "..."
+}
+```
+
+### Version History
+`wadm.api.{lattice}.model.versions.{name}`
+
+Retrieves a list of all stored versions of a given model, including when that version was created. 
+
+**Request**: empty
+
+**Response**: 
+```json
+[
+    {
+        "version": "...",
+        "created": "RFC3339",
```

Comment:
Can you wrap this in something like `<RFC3339 Timestamp>` or the like to make it clear that this is the desired format?